index.html

```diff
@@ -18,6 +18,12 @@
                     /_/                       
     </pre>
 
+    <div class="navbar">
```

Comment:
It might be useful to link to all the sections of your page in this section instead to just these two.